src/persist/src/indexed/encoding.rs

```diff
@@ -1174,26 +1162,6 @@ mod tests {
         };
         assert_eq!(b.validate(), Ok(()));
 
-        // Trace since at nonzero trace seal
```

Comment:
let's keep these tests but now verify that they succeed

Reply:
:+1: